docs/nodes/validate/add-a-validator.md

```diff
@@ -48,182 +48,304 @@ The response has your node’s ID:
+To create your own start times, please follow the steps below:
+
+Locate this line in the file
+
+```ts
+const startTime: BN = UnixNow().add(new BN(60 * 1));
+const endTime: BN = startTime.add(new BN(26300000));
+```
+
+Change `startTime` and `endTime` to new `BN` values, for example:
+
+```ts
+const startTime: BN = new BN(1654656829); // Wed Jun 08 2022 02:53:49 GMT+0000
+const endTime: BN = new BN(1662602029); // Thu Sep 08 2022 01:53:49 GMT+0000
 ```
 
-`startTime` and `endTime`
+#### Delegation Fee Rate
 
-When one issues a transaction to join the Primary Network they specify the time they will enter (start validating) and leave (stop validating.) The minimum duration that one can validate the Primary Network is 2 weeks, and the maximum duration is one year. One can re-enter the Primary Network after leaving, it’s just that the maximum _continuous_ duration is one year. `startTime` and `endTime` are the Unix times when your validator will start and stop validating the Primary Network, respectively. `startTime` must be in the future relative to the time the transaction is issued.
+Avalanche allows for delegation of stake. This parameter is the percent fee this validator charges when others delegate stake to them. For example, if `delegationFeeRate` is `10` and someone delegates to this validator, then when the delegation period is over, 10% of the reward goes to the validator and the rest goes to the delegator, if this node meets the validation reward requirements.
 
-`stakeAmount`
+#### Stake Amount
 
-In order to validate the Primary Network, one must stake AVAX. This parameter defines the amount of AVAX staked.
+The sample code assigns the variable, `stakeAmount` to be the minimum amount of AVAX that a node must stake to become a validator(1 AVAX).
+
+```ts
+const stakeAmount: any = await pchain.getMinStake();
+```
 
-`rewardAddress`
+To stake a different amount, please replace the code above with a stake amount of 1 AVAX or greater. Below sets a new value of 10,000 AVAX (`10000000000000` GWEI). Value is set in GWEI format where `1,000,000,000` GWEI = 1 AVAX.
 
-When a validator stops validating the Primary Network, they will receive a reward if they are sufficiently responsive and correct while they validated the Primary Network. These tokens are sent to `rewardAddress`. The original stake will be sent back to an address controlled by `username`.
+Please follow the example below for steps to change the stake amount:
+
+Locate this line in the file
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    stakeAmount.minValidatorStake,
+    ...
+  )
+```
+
+and change the default value to a new `BN` value, for example:
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    new BN(1e12),
+    ...
+  )
+```
+
+#### Addresses
+
+`toAddresses`
+
+An array of addresses as [Buffer](https://github.com/feross/buffer) who receive the staked tokens at the end of the staking period.
```

Comment:
Please provide example of using [Buffer](https://github.com/feross/buffer) if that is the case.

Reply:
I realize this is confusing. I believe that it would be better to just remove the mention of `Buffer` as it pertains more to how data is parsed and doesnt have an effect on how addresses are passed in.